server/utils.js

```diff
@@ -464,44 +465,47 @@ function dataToSchema(listOfElements) {
 }
 
 function fixAddress(schemaObject, geoObject) {
+  // The returned object will be POST/PATCH(ed),
```

Comment:
various comments here:

the name of the function seems to be misleading fix??
on the other hand you can use options=keyValues in updates as well